src/app/app.component.scss

```diff
@@ -0,0 +1,54 @@
+@import url('https://fonts.googleapis.com/css2?family=ABeeZee&family=Comfortaa:wght@300;400;500;600;700&display=swap');
+
+.navigation{
+    display: flex;
+    flex-direction: row;
+    width: auto;
+    background-color: #141414;
+    height: 40px;
+    margin-bottom: 10px;
+    border-bottom-right-radius: 10px;
+    align-items: center;
+
+    & span{
+        margin-left: 15px;
+        font-size: 20px;
+        margin-right: 30px; 
+        color: #5998C5;
+        font-family: ABeeZee;
+    }
+
+    & a{
+        margin: 0px 10px;
+        font-size: 16px; 
+        color: white;
+        text-decoration: none;
+        transition-duration: .3s;
+        font-family: ABeeZee;
```

Comment:
define variables for fonts, colors, font-sizes, etc globally, import the files and use them in files. 

Remember the DRY principles, "Don't repeat yourself"